controllers/drpolicy_controller_test.go

```diff
@@ -283,10 +283,53 @@ var _ = Describe("DrpolicyController", func() {
 					},
 				)
 			}()
-			drpolicy.Spec.SchedulingInterval = `3s`
-			Expect(k8sClient.Create(context.TODO(), drpolicy)).To(MatchError(err))
-			drpolicy.Spec.SchedulingInterval = `0`
-			Expect(k8sClient.Create(context.TODO(), drpolicy)).To(MatchError(err))
+			drp := drpolicy.DeepCopy()
+			drp.Spec.SchedulingInterval = `3s`
+			Expect(k8sClient.Create(context.TODO(), drp)).To(MatchError(err))
+			drp.Spec.SchedulingInterval = `0`
+			Expect(k8sClient.Create(context.TODO(), drp)).To(MatchError(err))
+		})
+	})
+
+	When("a drpolicy is having nil in DRCluster", func() {
```

Comment:
May be you could rephrase the when clause a bit here something like "a drpolicy without any DRCluster" or no DRCluster by replacing the nil.

Reply:
I have rephrased it.